opentelemetry-journald-logs/src/lib.rs

```diff
@@ -0,0 +1,292 @@
+use async_trait::async_trait;
+use libc::{c_int, c_void};
+use opentelemetry::logs::AnyValue;
+use std::ffi::CString;
+use std::fmt::Debug;
+
+use opentelemetry::logs::LogError;
+use opentelemetry_sdk::export::logs::{LogData, LogExporter};
+
+use opentelemetry::logs::Severity;
+
+#[cfg(feature = "json")]
+use serde::Serialize;
+
+extern "C" {
+    fn sd_journal_sendv(iov: *const libc::iovec, n: libc::c_int) -> libc::c_int;
+}
+
+#[derive(Default)]
+pub struct JournaldLogExporterBuilder {
+    identifier: Option<String>,
+    message_size_limit: Option<usize>,
+    attribute_prefix: Option<String>,
+    json_format: bool,
+}
+
+impl JournaldLogExporterBuilder {
+    pub fn identifier(mut self, identifier: &str) -> Self {
+        self.identifier = Some(identifier.to_string());
+        self
+    }
+
+    pub fn message_size_limit(mut self, message_size_limit: usize) -> Self {
+        self.message_size_limit = Some(message_size_limit);
+        self
+    }
+
+    pub fn attribute_prefix(mut self, attribute_prefix: Option<String>) -> Self {
```

Comment:
It would be more user friendly to just ask them for a string slice instead of having them to call `to_string()` and wrapping it with an `Option`.

```suggestion
    pub fn attribute_prefix(mut self, attribute_prefix: &str) -> Self {
```

Reply:
Done